gensim/models/nmf.py

```diff
@@ -146,9 +225,8 @@ def get_topics(self, normalize=None):
     def __getitem__(self, bow, eps=None):
         return self.get_document_topics(bow, eps)
 
-    def show_topics(self, num_topics=10, num_words=10, log=False,
-                    formatted=True, normalize=None):
-        """Get a representation for selected topics.
+    def show_topics(self, num_topics=10, num_words=10, log=False, formatted=True, normalize=None):
+        """Get the topics sorted by sparsity.
```

Comment:
What does `sparsity` mean here? The code looks like it's computing a column-wise average (?).

Reply:
Right, fixed that.